apps/web/src/pages/_app.tsx

```diff
@@ -56,7 +56,11 @@ export default function App({
   OpenAPI.CREDENTIALS = 'include';
   OpenAPI.WITH_CREDENTIALS = true;
 
-  const excludeLayoutPaths = ['/signin', '/register'];
+  const excludeLayoutPaths = [
+    '/signin',
+    '/register',
+    '/create-template/success',
```

Comment:
is there a reason we need this? this should only be for URIs that don't need a user to be logged in.

Reply:
Oh I had done that because it doesn't have the side bar.  Would it be better if I kept it in the layout and overlayed the background color over the navbar instead?

Reply:
oh nvm im stupid lol that works